### PR TITLE
Added break and continue statements , updated python.md

### PR DESCRIPTION
**Pull request by arshanand1**
I have added break and continue statements with the explanation of their working by giving the example of a for loop.

### CODE REVIEW

**Comment by arshanand1** [comment]
Fix Bug #148

File: cpp.md [hunk 1]
```diff
@@ -168,6 +168,28 @@ default:
  //code to be executed when all the above cases are not matched;    
 } 
 ```
+## Break and Continue Statements
+
+### 1. Break
+```c
+for(i=0; i<5; i++){
+   if(i==4){
```
Comment (onecompiler-ops) [inline cpp.md:176]: can you add more realistic example

File: cpp.md [hunk 2]
```diff
@@ -168,6 +168,28 @@ default:
...
+      }
+}
+```
+The break statement helps in coming out from the loop when your condition is satisfied.
+
+### 2. Continue
+```c
+for(i=0; i<5; i++){
+   if(i==4){
+      continue;
```
Comment (onecompiler-ops) [inline cpp.md:187]: can you add more realistic example

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

File: cpp.md (hunk 1 above)
Comment (arshanand1) [reply cpp.md:176]: for break statement should i use an example of switch statement? that seems to be a more realistic one

File: cpp.md (hunk 2 above)
Comment (arshanand1) [reply cpp.md:187]: Can you please suggest me something like should i display the output of the code as well to explain it better or if any other example of code you think which can explain continue statement better ?

Comment (kshitijmishra27) [reply cpp.md:187]: @arshanand1  i think you can also include comments in your code it will make your code understandable for the people

Comment (arshanand1) [reply cpp.md:187]: @kshitijmishra27 For sure! Thanks for the suggestion.

**Comment by arshanand1** [comment]
changes that were asked have been made.

File: cpp.md (hunk 2 above)
Comment (onecompiler-ops) [reply cpp.md:211]: 1. Keep it precise 
2. Add a real-world examples i.e where break, continue are really useful and you can't solve without them

**Review by onecompiler-ops — CHANGES_REQUESTED** [review]

File: cpp.md (hunk 2 above)
Comment (arshanand1) [reply cpp.md]: Real life examples added to the file cpp.md

**Comment by onecompiler-ops** [comment]
lifted from here https://www.folkstalk.com/2022/10/break-with-while-loop-with-code-examples.html

**Comment by arshanand1** [comment]
let me add another code ?
You weren't accepting my code's explanation

**Comment by onecompiler-ops** [comment]
@arshanand1 Please produce original and meaningful content to get the PRs accepted

**Pull request closed without merging**